google/cloud/dns/client.py

```diff
@@ -91,14 +91,23 @@ def quotas(self):
 
         :rtype: mapping
         :returns: keys for the mapping correspond to those of the ``quota``
-                  sub-mapping of the project resource.
+                  sub-mapping of the project resource. ``kind`` is stripped
+                  from the results.
```

Comment:
Thanks for adding this to the comment. It seems we always stripped this away silently :)